src/eko/evolution_operator/grid.py

```diff
@@ -39,40 +46,20 @@ class OperatorGrid(sv.ModeMixin):
 
     def __init__(
         self,
-        mu2grid: npt.NDArray,
-        order: tuple,
-        masses: tuple,
+        mu2grid: List[EPoint],
+        order: Order,
+        masses: List[float],
         mass_scheme,
+        thresholds_ratios: List[float],
         intrinsic_flavors: list,
         xif: float,
         n3lo_ad_variation: tuple,
         configs: Configs,
         debug: Debug,
-        thresholds_config,
-        couplings,
-        interpol_dispatcher,
+        atlas: Atlas,
+        couplings: Couplings,
+        interpol_dispatcher: InterpolatorDispatcher,
     ):
-        """Initialize `OperatorGrid`.
```

Comment:
In principle both, but the whole content of this file is more or less deprecated, since it is the *old runner*, and it is not used any longer by the *new* runner.

I would skip the docstrings update, just because they were outdated even before this PR, and in principle it is not this PR business to fix them (but to supersede). However, if you'd like to do it, feeel free.